framework/test/mocha/integration/blocks/chain/delete_last_block_db_trs.js

```diff
@@ -151,16 +153,16 @@ describe('system test (blocks) - chain/popLastBlock', () => {
 					library.logic.account.merge = merge;
 					done();
 				});
-
```

Comment:
Why are these tests failing?

Reply:
The skipped ones are failing as they're expecting `account.merge()` error messages that are not the same returned by process transaction